Add strap linking to GitHub repo on a timeout unless hovered over

grmbyrn opened the pull request:
I've added a new SvelteKit component, called `Strap`, to enhance the homepage by displaying a link to the Torrust GitHub repository at the top of the home page. The purpose is to make this link visible for a limited time, encouraging users to explore our repository, but also going away if they aren't currently interested.

## Strap Component Highlights:
- Display Logic: The `Strap` component includes logic to display the GitHub link for 10 seconds. Hovering over the component stops the timer.
- Integration in Header Component: The `Strap` component is integrated into the `Header` component and is conditionally rendered on the homepage.

## How to Spot the Changes:
- In `Strap.svelte`, you'll find the logic for the timed display and the associated HTML and CSS.
- In `Header.svelte`, the integration of the `Strap` component is wrapped within a conditional check for the homepage.

## Why this Change?
This enhancement is aimed at increasing visibility and encouraging engagement with our GitHub repository. The `Strap` component adds a subtle yet effective call-to-action on the homepage.

Commit:
Add strap linking to GitHub repo on a timeout unless hovered over

josecelano reviewed (changes requested):
Hi @grmbyrn cool!

I only see two problems:

- The link. We have more repos, not only the tracker. Maybe we can link to the organization.
- We could need the top part of the page for announcements in the future. But I guess we could put them on top of this element.

It's OK but I would change the link.

grmbyrn commented:
> Hi @grmbyrn cool!
> 
> I only see two problems:
> 
> * The link. We have more repos, not only the tracker. Maybe we can link to the organization.
> * We could need the top part of the page for announcements in the future. But I guess we could put them on top of this element.
> 
> It's OK but I would change the link.

No problem, @josecelano. Just before I change it, do you mean link it to this: https://github.com/torrust?

With future announcements at the top of the page, this can be removed easily, covered, or an interval/conditional could be set depending on what we want to do with it.

josecelano commented:
> > Hi @grmbyrn cool!
> > I only see two problems:
> > 
> > * The link. We have more repos, not only the tracker. Maybe we can link to the organization.
> > * We could need the top part of the page for announcements in the future. But I guess we could put them on top of this element.
> > 
> > It's OK but I would change the link.
> 
> No problem, @josecelano. Just before I change it, do you mean link it to this: https://github.com/torrust?
> 
> With future announcements at the top of the page, this can be removed easily, covered, or an interval/conditional could be set depending on what we want to do with it.

Hi @grmbyrn I think "https://github.com/torrust" would be fine.

grmbyrn committed:
Update Strap.svelte

Change href in link from https://github.com/torrust/torrust-tracker to https://github.com/torrust

grmbyrn commented:
> > > Hi @grmbyrn cool!
> > > I only see two problems:
> > > 
> > > * The link. We have more repos, not only the tracker. Maybe we can link to the organization.
> > > * We could need the top part of the page for announcements in the future. But I guess we could put them on top of this element.
> > > 
> > > It's OK but I would change the link.
> > 
> > 
> > No problem, @josecelano. Just before I change it, do you mean link it to this: https://github.com/torrust?
> > With future announcements at the top of the page, this can be removed easily, covered, or an interval/conditional could be set depending on what we want to do with it.
> 
> Hi @grmbyrn I think "https://github.com/torrust" would be fine.

No worries @josecelano. I made the href change.

josecelano commented:
> git pull git@github.com:grmbyrn/torrust-website.git strap

Hi @grmbyrn I think it should be: "Watch our [GitHub repos](https://github.com/torrust) to stay up to date" in plural :-)

grmbyrn committed:
Update Strap.svelte

Change 'repo' to 'repos'

grmbyrn commented:
Hi @josecelano, that makes sense. I've changed it to the plural.

josecelano commented:
ACK 6496944d8d3ad2aea39c19395b410ebefd454076
